docs/units-and-global-variables.rst

```diff
@@ -161,6 +161,13 @@ Members of string
 
 - ``string.concat(...) returns (string memory)``: :ref:`Concatenates variable number of string arguments to one string array<string-concat>`
 
+.. index:: event members
```

Comment:
This section is about global variables. You need to add this documentation to the type documentation for events, or just to the documentation for events in general.